apps/charterafrica/server.ts

```diff
@@ -7,6 +7,9 @@ import next from "next";
 import nodemailerSendgrid from "nodemailer-sendgrid";
 import payload from "payload";
 import { Payload } from "payload/dist/payload";
+import dotenv from "dotenv";
+
+dotenv.config({ path: path.resolve(__dirname, "./.env.local") });
```

Comment:
@koechkevin Not sure why we are hardcoding `.env.local` here since its only meant for local environments.

Reply:
`loadEnvConfig` below this line should load all necessary `.env` files. We SHOULDN'T need this line.